### PR TITLE
Release threshold folder imports nit

### PR DESCRIPTION
**Pull request by nhsiehgit**
Cleans up release threshold model imports & adds related name to the relase_threshold table for projects

### CODE REVIEW

File: src/sentry/api/endpoints/release_threshold_details.py [hunk 1]
```diff
@@ -10,8 +10,7 @@
 from sentry.api.bases.project import ProjectEndpoint
 from sentry.api.exceptions import ResourceDoesNotExist
 from sentry.api.serializers import serialize
-from sentry.models.project import Project
-from sentry.models.release_threshold.releasethreshold import ReleaseThreshold
```
Comment (cathteng) [inline src/sentry/api/endpoints/release_threshold_details.py:14]: where do we prefer importing models like this? i see both styles sometimes

**Review by cathteng — COMMENTED** [review]
lgtm, but maybe get confirmation from someone from migrations?

File: src/sentry/api/endpoints/release_threshold_details.py (hunk 1 above)
Comment (nhsiehgit) [reply src/sentry/api/endpoints/release_threshold_details.py:14]: Specific imports are fine, however we're already exporting everything from the models/__init__.py so imo it's cleaner to simply have the top level path for imports.

File: src/sentry/api/serializers/models/release_threshold.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 from sentry.api.serializers import Serializer, register, serialize
+from sentry.models import ReleaseThreshold
```
Comment (JoshFerge) [inline src/sentry/api/serializers/models/release_threshold.py:2]: should favor importing directly from the model file. (i.e. import from `sentry.models.releasethreshold`). We should work towards enforcing this somewhere

File: src/sentry/api/endpoints/release_threshold_details.py (hunk 1 above)
Comment (evanpurkhiser) [reply src/sentry/api/endpoints/release_threshold_details.py:14]: I think we're trying to get away from this like @JoshFerge pointed out

Comment (nhsiehgit) [reply src/sentry/api/endpoints/release_threshold_details.py:14]: Ah - sounds like we're actually pushing for the explicit imports so reverting and changing to that

**Review by JoshFerge — COMMENTED** [review]
could we isolate the migration from the nits? try in general to keep migration PRs separate from other changes.

**Review by JoshFerge — APPROVED** [review]

**Pull request merged**